lib/primitives/mtx.js

```diff
@@ -1340,6 +1340,50 @@ class MTX extends TX {
     return select;
   }
 
+  /**
+   * Select and add coins to fill the transaction
+   * and set the output value to the total value minus fee
+   * @param {Coin[]} coins - list of coins to fill with
+   * @param {Object?} options - fill options
+   * @param {(Amount)?} options.threshold - highest coin value
+   * in satoshis that should be included in sweep
+   * @param {(Rate)?} options.rate - rate to pay for tx fee
+   * @returns {MTX} mtx - returns a filled mtx
+   */
+   async fill(coins, options) {
+    assert(coins.length, 'Must pass in coins to fill');
+    assert(options, 'Options are required');
+    assert(this.outputs.length, 'Must have at least 1 output');
+
+    // must set option to fill for coin selection
+    if (!options.fill)
+      options.fill = true;
+    // unless otherwise set, we want to sort and selct coins by value
+    // fill will automatically sort from lowest to highest
+    if (!options.select)
```

Comment:
So do we want to accept other sorting options ?

Reply:
Yeah, I wasn't sure about this. I guess it adds extra complexity to have the ability to pass another option but the requirement for now was just default to reverse value sorting